x-pack/solutions/security/packages/security-ai-prompts/src/get_prompt.test.ts

```diff
@@ -254,7 +277,7 @@ describe('get_prompt', () => {
         connectorId: 'connector-123',
       });
 
-      expect(result).toBe('Hello world this is a system prompt for bedrock claude-3-5-sonnet');
+      expect(result).toBe('Hello world this is a system prompt for bedrock claude-3-7-sonnet');
```

Comment:
@elastic/security-generative-ai due to the update of the rainbow sprinkles model, had to adapt this test accordingly